src/Definition/Teradata.php

```diff
@@ -336,7 +338,8 @@ public function getSQLDefinition()
         $type = $this->getType();
         $definition = $type;
         if (!in_array($definition, self::TYPES_WITHOUT_LENGTH)) {
-            $length = $this->getLength() ?: $this->getDefaultLength();
+            $length = $this->getLength();
+            $length = is_null($length) ? $this->getDefaultLength() : $length;
```

Comment:
porovnání je o tick rychlejší než funkce, takže je to vlastně fuk :)
```suggestion
            $length = $length === null ? $this->getDefaultLength() : $length;
```